jit-service/src/main/java/org/dcsa/jit/mapping/VesselMapper.java

```diff
@@ -27,6 +33,7 @@ public interface VesselMapper {
   @AfterMapping
   default void mapVesselOperatorCarrier(
     Vessel vessel, @MappingTarget VesselTO.VesselTOBuilder vesselTOBuilder) {
+    if(vessel.getVesselOperatorCarrier() == null) return;
```

Comment:
As a principle I strongly prefer that if's have brackets and linebreaks even in a simple case like this